apps/mobile/ios/Podfile

```
@@ -17,6 +17,131 @@ install! 'cocoapods',
+      # File.delete(zip_path) if File.exist?(zip_path)
+      
+      puts "gopenpgp v#{version} downloaded, compiled and moved successfully"
+    else
+      puts "Error: Failed to compile gopenpgp library"
+    end
+      FileUtils.rm_rf(download_dir) if Dir.exist?(download_dir)
+      puts "Cleaned up download directory"
+  else
+    puts "gopenpgp v#{version} already exists, skipping download"
+  end
+end
```

RuboCop (1.80.2)

[convention] Assignment Branch Condition size for `download_and_compile_gopenpgp` is too high. [<10, 45, 13> 47.9/23]

(Metrics/AbcSize)